ibis/sql/postgres/tests/conftest.py

```diff
@@ -19,9 +19,6 @@
 
 import ibis
 
-pytest.importorskip('sqlalchemy')
-pytest.importorskip('psycopg2')
```

Comment:
Interesting. Those should be moved to `postgresql/test_functions.py` then. We still need to skip if these aren't installed.